### PR TITLE
Add st2workflowengine logging conf to appropriate st2 conf files

### PR DESCRIPTION
**Pull request by m4dcoder**
Add the st2workflowengine logging configuration path to the st2.conf for packaging and production.

### CODE REVIEW

**Comment by blag** [comment]
LGTM :+1:

**Comment by m4dcoder** [comment]
@blag You need to approve this not just comment LGTM.

File: conf/st2.package.conf [hunk 1]
```diff
@@ -77,3 +77,6 @@ url = amqp://guest:guest@127.0.0.1:5672/
 
 [ssh_runner]
 remote_dir = /tmp
+
+[workflow_engine]
```
Comment (armab) [inline conf/st2.package.conf]: `rulesengine` vs `workflow_engine` inconsistency makes me :cry: 

I think that's what @lakshmi-kannan mentioned a few days ago.

Comment (Kami) [reply conf/st2.package.conf]: Same here.

There are similar examples in other places. I personally prefer `_`.

**Pull request merged**